bitnami/nats/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.podDisruptionBudget }}
```

Comment:
Here as well, you may want to use the standard approach `.Values.pdb.create` (see my comment below)